api/editor/api-editor/src/main/java/ca/bc/gov/tno/keycloak/KeycloakConfig.java

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.tno.keycloak;
+
+import org.keycloak.adapters.springboot.KeycloakSpringBootConfigResolver;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration settings Keycloak.
+ */
+@Configuration
+class KeycloakConfig {
+
+  @Bean
+  public KeycloakSpringBootConfigResolver KeycloakConfigResolver() {
```

Comment:
Dependency update required separating the configuration... Spring is a piece of junk.